Assets/MixedRealityToolkit.Providers/WindowsMixedReality/XR2018/WindowsMixedRealityDeviceManager.cs

```diff
@@ -80,7 +80,11 @@ public bool CheckCapability(MixedRealityCapability capability)
             }
             else // Pre-Windows 10 1903.
             {
+                // Ensure compatibility with the pre-2019.3 XR architecture for customers / platforms
+                // with legacy requirements.
+#pragma warning disable 0618
```

Comment:
I'm about to open a PR (or you can add it here) with a new csc file next to this assembly, removing the warning for the whole assembly. There's a lot more than just this one in here

Reply:
![image](https://user-images.githubusercontent.com/3580640/73478613-40faae00-434b-11ea-9995-7c058d3d9808.png)